site_scons/site_tools/md2html.py

```diff
@@ -14,12 +14,10 @@
 import SCons.Environment
 
 DEFAULT_EXTENSIONS = frozenset({
-	# Supports tables, HTML mixed with markdown, code blocks and more
+	# Supports tables, HTML mixed with markdown, code blocks, custom attributes and more
 	"markdown.extensions.extra",
-	# Allows TOC with [TOC], allows anchors set by "# Title {#foo}"
+	# Allows TOC with [TOC]"
 	"markdown.extensions.toc",
-	# Allows setting attributes with {@id=foo}
-	"markdown.extensions.legacy_attrs",

```

Comment:
this has been removed as `extra` contains abetter method of setting attributes, and this method with `@` conflicts with `mdx_gh_links` for github username links.